packages/editor/src/client/api/index.ts

```diff
@@ -646,7 +646,7 @@ export type MemberPlan = {
   slug: Scalars['String'];
   image?: Maybe<Image>;
   description?: Maybe<Scalars['RichText']>;
-  tags: Array<Scalars['String']>
+  tags?: Maybe<Array<Scalars['String']>>;
```

Comment:
Is this change really necessary and when yes why?

Reply:
It was causing errors compiling . I think others have run into the same, but should probably be fixed somewhere else.